File: ruby/buclefor.rb
# Definimos un arreglo de nombres
nombres = ['camila', 'yury', 'cristian', 'sebastian']

# Usamos un bucle para imprimir cada nombre en el arreglo
nombres.each do |nombre|
  puts nombre
end

# Definimos un arreglo vacío para almacenar personas
personas = []

# Creamos un hash para cada persona con nombre y edad
a = {'nombre' => 'camila', 'Edad' => 22}
b = {'nombre' => 'yury', 'Edad' => 28}
c = {'nombre' => 'cristian', 'Edad' => 31}
d = {'nombre' => 'sebastian', 'Edad' => 26}

# Agregamos cada hash al arreglo personas
personas.push(a)
personas.push(b)
personas.push(c)
personas.push(d)

# Usamos un bucle para imprimir el nombre y la edad de cada persona
personas.each do |persona|
  puts "#{persona['nombre']} #{persona['Edad']}"
end

File: php/buclefor.php
<?php
$nombres = ['camila', 'yury', 'cristian','sebastian \n\n'];

foreach ($nombres as $nombre) {
    echo $nombre . "\n";
}

$personas = [];

$a = ['nombre' => 'camila' , 'edad' => 22];
$b = ['nombre' => 'yury' , 'edad' => 28];
$c = ['nombre' => 'cristian' , 'edad' => 31];
$d = ['nombre' => 'sebastian' , 'edad' => 26];

array_push ($personas, $a, $b, $c, $d);

foreach ($personas as $persona) {
    echo $persona['nombre']. " " . $persona['edad'] . "\n";
}
?>

